nss-day-cohort-60/national-parks-django-api
# Jd user login/registration refactoring

Added auth_view to log in and register users.

db usernames should be equal to user emails and all passwords reset to 'gandalfsucks12' on 1st login attempt

Testing:
1. switch to JD-login client branch
2. switch to JD-user-passwords-fix server branch, restart debugger.
3. check client side login/registration functions work per https://github.com/nss-day-cohort-60/national-parks-client-v2/pull/3 client side pull request
4. confirm users in auth_user as expected with proper staff status.
